packages/analytics-browser/package.json

```diff
@@ -33,7 +33,7 @@
     "publish": "node scripts/publish/upload-to-s3.js",
     "test": "jest",
     "typecheck": "tsc -p ./tsconfig.json",
-    "version": "yarn build",
+    "version": "GENERATE_SNIPPET=true yarn build",
```

Comment:
this is the second call which is called via lerna version. the first one is `yarn build` called directly from cd.yml.